include/TabWidget.h

```diff
@@ -59,6 +59,7 @@ class TabWidget : public QWidget
 	Q_PROPERTY( QColor tabText READ tabText WRITE setTabText)
 	Q_PROPERTY( QColor tabTitleText READ tabTitleText WRITE setTabTitleText)
 	Q_PROPERTY( QColor tabSelected READ tabSelected WRITE setTabSelected)
+	Q_PROPERTY( QColor tabTextSelected READ tabTextSelected WRITE setTabTextSelected)
```

Comment:
```suggestion
	Q_PROPERTY(QColor tabTextSelected READ tabTextSelected WRITE setTabTextSelected)
```

Reply:
Although the rest use `READ` and `WRITE`, it's also possible to use `MEMBER`:
```suggestion
	Q_PROPERTY(QColor tabTextSelected MEMBER m_tabTextSelected)
```

Reply:
Spaces fixed with commit 0fa18d405e7 where I have fixed style issues for all of `TabWidget`. I have kept the `READ` and `WRITE` versions due to consistency with the other properties.